packages/asc-core/src/components/ContextMenu/ContextMenuItem.ts

```diff
@@ -0,0 +1,31 @@
+import styled from '../../styled-components'
+import { ContextMenu } from './types'
+import Icon from '../Icon'
+import { getColor, getTypography } from '../../utils/themeUtils'
+
+const ContextMenuItem = styled.li<ContextMenu.ContextMenuItemProps>`
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 36px;
+  width: 100%;
+  cursor: pointer;
+  font-size: ${({ theme }) => getTypography(theme).fontWeightRegular};
+
+  ${Icon} {
+    margin: 5px 12px;
+  }
+
+  ${({ divider, theme }) =>
+    divider && `border-bottom: 1px solid ${getColor(theme, 'tint', 'level4')}`}
+
+  &:hover,
+  &:focus {
+    outline: none;
+    background-color: ${({ theme }) => getColor(theme, 'tint', 'level2')};
```

Comment:
The hover state for links in lists should be a transparent background and primary red for the text colour.

Reply:
This component  is based upon the selection dropdown, as there is no default for lists like this yet. So current setup is correct https://designsystem.amsterdam.nl/7awj1hc9f/p/6561a5